cfme/tests/cloud/test_providers.py

```diff
@@ -424,7 +424,7 @@ def test_api_port_max_character_validation_cloud(appliance):
 
 
 @pytest.mark.tier(2)
-@pytest.mark.uncollectif(lambda provider: not provider.one_of(AzureProvider))
+@pytest.mark.provider([AzureProvider], scope="function", override=True)
```

Comment:
uncollecting against Azure? Now collecting against it?

Reply:
uncollecting against _not_ azure, now marking for azure.

Reply:
This is uncollecting only if the provider is not one of Azure. I think it is more direct to simply mark the test with the Azure provider